automation/jenkins_build.sh

```diff
@@ -12,14 +12,12 @@ docker pull resin/${ARCH}-supervisor:${ESCAPED_BRANCH_NAME} || docker pull resin
 make SUPERVISOR_VERSION=${VERSION} JOB_NAME=${JOB_NAME} test-gosuper
 
 MAKE_ARGS="ARCH=${ARCH} \
-    JOB_NAME=${JOB_NAME} \
-    BASE_DISTRO=${BASE_DISTRO} \
     PUBNUB_SUBSCRIBE_KEY=${PUBNUB_SUBSCRIBE_KEY} \
     PUBNUB_PUBLISH_KEY=${PUBNUB_PUBLISH_KEY} \
     MIXPANEL_TOKEN=${MIXPANEL_TOKEN}"
 
-make ${MAKE_ARGS} \
-    lint
+# Disabled until this is merged in npm https://github.com/npm/npm/pull/13257
```

Comment:
Why is this an issue, I can't see why the `make lint` command would hit the different devices/filesystems issues?